library/build.gradle

```diff
@@ -8,23 +8,25 @@ buildscript {
     repositories {
         jcenter()
         maven { url 'https://maven.google.com' }
+        google()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.0.0'
         classpath "org.jfrog.buildinfo:build-info-extractor-gradle:4.5.4"
     }
 }
 
-android {
-    compileSdkVersion 25
-    // gradle plugin > 3.0.0 uses its own version
-//    buildToolsVersion "26.0.2"
+repositories {
+    google()
+}
 
+android {
+    compileSdkVersion 28
     defaultConfig {
-        minSdkVersion 17
-        targetSdkVersion 22
+        minSdkVersion 23
```

Comment:
should we really increase the min sdk version? have there been any changes that require the lib to not be used on lower sdk versions? it is a pity to cut off older versions when developing a lib, if not absolutely necessary.